src/app/modules/OpenshiftStreams/dialogs/CreateInstance/api.ts

```diff
@@ -204,6 +204,10 @@ export const useCreateInstance = (): OnCreateKafka => {
           case ErrorCodes.INSUFFICIENT_QUOTA:
             onError('over-quota');
             break;
+          case ErrorCodes.REACHED_MAX_LIMIT_ALLOWED_KAFKA:
+          case ErrorCodes.INSTANCE_TYPE_NOT_SUPPORTED:
```

Comment:
FYI - While it is ok to have this error code this case might not happen as we should filter our regions that are not available for selection. If you are able to handle this case it means that we have bug in filtering of provider region dropdown

Reply:
Yes. It is handled properly but better to be safe and handle it here as well.

Reply:
Is this error code specific to regions? When I look at the list of error codes, I see a separate one for regions (https://github.com/redhat-developer/app-services-sdk-js/blob/main/packages/kafka-management-sdk/src/errors.ts#L64). 

On the topic of regions not being available, I think there are 3 cases that we could handle:
1. None of the regions are available - display an inline warning message on form load
    > ⚠️ Kafka instances are unavailable for creation
    > Kafka instances are no longer available. Try again in a few hours.
2. Some of the regions are available - display the unavailable regions as disabled. 
    - I consider this a short-term easy solution. As part of the redesign, it would be good to find a more user-friendly solution for this.
3. One or more regions were available on form load, but the selected region is no longer available on submit - display an inline error message
    > ❗Kafka instance can't be created
    > Kafka instances are no longer available. Try again in a few hours.

Reply:
Yes. Error code means that for instance type and region we reached our maximum capacity. In practice that happens only for kafka trials right now.

Very good rundown of use cases. Makes sense